src/index.tsx

```diff
@@ -28,6 +36,7 @@ export { useLogoutFunction } from "./hooks/useLogoutFunction"
 export { RedirectToLogin, RedirectToSignup, useRedirectFunctions } from "./hooks/useRedirectFunctions"
 export type { RedirectProps } from "./hooks/useRedirectFunctions"
 export { RequiredAuthProvider } from "./RequiredAuthProvider"
+export type {} from "./user"
```

Comment:
out of curiosity, why export an empty type?

Reply:
bad combination of copilot and reformating :)